internal/pkg/api/server/templates/dependencies/dependencies.go

```diff
@@ -31,38 +28,32 @@ type Container interface {
 	WithCtx(ctx context.Context, cancelFn context.CancelFunc) Container
 	PrefixLogger() log.PrefixLogger
 	RepositoryManager() (*repository.Manager, error)
-	DefaultRepository() model.TemplateRepository
+	Repositories() []model.TemplateRepository
```

Comment:
Premenovane, + zo `service.go` zmazane `repositories`.
Tj. `dependencies` daj mi zoznam repositories.
A aktualne to vrati `v.defaultRepositories`, ale do buducna bude mozne pridat repositories per project, tj. k `defaultRepositories` sa v tomto subore pridaju dalsie.